README.md

```diff
@@ -20,20 +20,11 @@ JsonLD supports [framing](http://json-ld.org/spec/latest/json-ld-framing/)
 Installation
 ------------
 
-The easiest way to use JsonLD is to integrate it as a dependency in your
-project's [composer.json](https://getcomposer.org/) file:
-
-```json
-{
-    "require": {
-        "ml/json-ld": "1.*"
-    }
-}
-```
-
-Installing is then a matter of running composer
+The easiest way to use JsonLD is to install `JsonLD` using [Composer](https://getcomposer.org/).
```

Comment:
Lots of repetition in that sentence. Could you please change it to:

```
The easiest way to install JsonLD is by requiring it with [Composer](https://getcomposer.org/)

    composer require ml/json-ld
```

Reply:
Hi @lanthaler, sorry for missing out your feedback! I applied your feedback and updated the branch with master.